packages/core-data/README.md

```diff
@@ -483,30 +483,30 @@ entity object if it exists and is received.
 
 _Parameters_
 
--   _state_ `Object`: State tree
+-   _state_ `State`: State tree
 -   _kind_ `string`: Entity kind.
 -   _name_ `string`: Entity name.
--   _key_ `number`: Record's key
--   _query_ `?Object`: Optional query.
+-   _key_ `RecordKey`: Record's key
+-   _query_ `EntityQuery< any >`: Optional query.
 
 _Returns_
 
--   `Object|undefined`: Record.
+-   `EntityRecord | undefined`: Record.
 
 ### getEntityRecordEdits
 
 Returns the specified entity record's edits.
 
 _Parameters_
 
--   _state_ `Object`: State tree.
+-   _state_ `State`: State tree.
 -   _kind_ `string`: Entity kind.
 -   _name_ `string`: Entity name.
--   _recordId_ `number`: Record ID.
+-   _recordId_ `RecordKey`: Record ID.
 
 _Returns_
 
--   `Object?`: The entity record's edits.
+-   `any | undefined`: The entity record's edits.
```

Comment:
this type doesn't make sense. I know it's supposed to communicate that there might be no return value, but `undefined` is already covered by `any`.